alica_engine/src/engine/Logger.cpp

```diff
@@ -23,7 +23,7 @@ Logger::Logger(AlicaEngine* ae) {
     supplementary::SystemConfig* sc = supplementary::SystemConfig::getInstance();
     this->active = (*sc)["Alica"]->get<bool>("Alica.EventLogging.Enabled", NULL);
     if (this->active) {
-        char buffer[50];
+        char buffer[1024];
```

Comment:
No, the buffer does not need to be this large, strftime produces a result with a fixed length.
What's more, neither buffer nor timeString are actually used, they can be removed entirely.